### ISSUE TITLE
paddlepaddle2.0版本，训练时验证出现错误

### ISSUE THREAD

**Issue by stu166** [PaddlePaddle/Paddle#29596]
train_dataset = PetDataset(mode='train') # 训练数据集
val_dataset = PetDataset(mode='test') # 验证数据集

optim = paddle.optimizer.RMSProp(learning_rate=0.001,
                                 rho=0.9,
                                 momentum=0.0,
                                 epsilon=1e-07,
                                 centered=False,
                                 parameters=model.parameters())
model.prepare(optim, paddle.nn.CrossEntropyLoss(), metrics=paddle.metric.Accuracy())
model.fit(train_dataset,
          val_dataset,
          epochs=10,
          batch_size=32,
          verbose=1)

结果如下：acc的精度不正常
step 65/65 [==============================] - loss: 0.4405 - acc: 152.8935 - 380ms/step        
Eval begin...
step 17/17 [==============================] - loss: 0.1708 - acc: 156.6081 - 371ms/step         
Eval samples: 518
Epoch 2/10
step 65/65 [==============================] - loss: 0.1953 - acc: 159.5491 - 381ms/step        
Eval begin...
step 17/17 [==============================] - loss: 0.0929 - acc: 163.6081 - 370ms/step         
Eval samples: 518
Epoch 3/10
step 65/65 [==============================] - loss: 0.2937 - acc: 168.6344 - 379ms/step        
Eval begin...
step 17/17 [==============================] - loss: 0.0793 - acc: 145.4768 - 373ms/step         
Eval samples: 518
Epoch 4/10
step 65/65 [==============================] - loss: 0.1390 - acc: 166.7717 - 381ms/step        
Eval begin...
step 17/17 [==============================] - loss: 0.0738 - acc: 150.1100 - 370ms/step         
Eval samples: 518
Epoch 5/10
step 65/65 [==============================] - loss: 0.1521 - acc: 159.7158 - 379ms/step        
Eval begin...
step 17/17 [==============================] - loss: 0.0700 - acc: 163.5849 - 372ms/step

**Comment by frankwhzhang** [comment]
请问是官方模型还是自己的模型

**Comment by stu166** [comment]
官方的模型，就是文档2.0rc里应用实践里的U-Net模型，数据是自己定义的数据。做的是图像分割，acc评价的值都是100多，显得不正常



---原始邮件---
发件人: "zhang wenhui"<notifications@github.com&gt;
发送时间: 2020年12月15日(周二) 下午3:50
收件人: "PaddlePaddle/Paddle"<Paddle@noreply.github.com&gt;;
抄送: "stu166"<806539041@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题: Re: [PaddlePaddle/Paddle] paddlepaddle2.0版本，训练时验证出现错误 (#29596)




 
请问是官方模型还是自己的模型
